setup.py

```diff
@@ -33,14 +25,10 @@
 
 long_description = open("README.rst").read()
 
-install_requires = ['pymongo>=3.10,<4']
+install_requires = ['pymongo>3.10,<4']
 
 tests_require = ['mockupdb>=1.4.0']
 
```

Comment:
I think we should add a guard to setup.py like this:
```
if sys.version_info[:2] < (3, 5):
    raise Exception("This version of Motor requires Python >=3.5")
```

This will cause installation to fail early on clients which don't support `python_requires`.

Reply:
Sounds good. Done.